README.md

```diff
@@ -83,9 +83,9 @@ pnpm add @strapi/sdk-js
 To interact with your Strapi backend, initialize the SDK with your Strapi API base URL:
 
 ```typescript
-import { createStrapiSDK } from '@strapi/sdk-js';
+import { strapiSDK } from '@strapi/sdk-js';
 
-const sdk = createStrapiSDK({ baseURL: 'http://localhost:1337/api' });
+const sdk = strapiSDK({ baseURL: 'http://localhost:1337/api' });
```

Comment:
Whats the reason for this rename? I actually prefer it before

Reply:
Mainly for two reasons
- less verbose
- more conventional

We were also thinking about directly exposing the class constructor, but that might be for a later iteration if needed